deployment/constructs/dynamodb-table.ts

```diff
@@ -36,7 +38,7 @@ export class DynamoDBTable extends Construct {
       streamable = true,
     } = props;
 
-    const tableName = `${stage}-${name}`;
+    const tableName = `${project}-${stage}-${name}`;
```

Comment:
this will not be easy to roll out across our apps, but I think it would be nice to have the project name at the beginning of resource names.